code/modules/client/client procs.dm

```diff
@@ -770,3 +773,15 @@
 			winset(src, "rpane.changelog", "background-color=#40628a;text-color=#FFFFFF")
 		else
 			winset(src, "rpane.changelog", "background-color=none;text-color=#000000")
+
+
+/client/proc/send_ssd_warning(mob/M)
+	if(!config.ssd_warning)
+		return FALSE
+	if(ssd_warning_acknowledged)
+		return FALSE
+	if(M && M.player_logged < 30) // Give it ~60 seconds, just in case they drop SSD during combat.
```

Comment:
It's probably better to make this a define on top of the document. Something like "SSD_WARNING_TIMER"

Reply:
Done, with #undef afterwards.